site/docs/components/pill/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Pill
+data:
+  description: Pill is a small visual element that can contain descriptive text and an icon. You can use Pill to label, tag or categorize content. With Pill, users can trigger actions, make selections or filter results.
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/main/packages/lab/src/pill-next/PillNext.tsx"
+  package:
+    name: "@salt-ds/lab"
+  alsoKnownAs: ["Tag", "Chip", "Badge"]
+  relatedComponents:
+    [
+      { name: "ToggleButton", relationship: "similarTo" },
+      { name: "Button", relationship: "similarTo" },
```

Comment:
maybe mention it may contain Icon?

Reply:
metadata. on the next line here

Reply:
Do we say it may contain Icon if it's the consumer which adds the icon, rather than it being built into the component?

Reply:
That's a very good point. Looking at the word docs template, it says:

[OPTIONAL] A list of other components that this component uses | “Form Field” may use the “Button” component


Maybe we need some more clarification on this?

Reply:
Also, Button component, which I believe should be the reference here, has Icon https://www.saltdesignsystem.com/salt/components/button/examples

Reply:
true, I've updated to add contains